content/english/java-tictactoe/placeholder.md

```diff
@@ -0,0 +1,101 @@
+# Read and Process Input
```

Comment:
Please add metadata for this page, and set draft: true

Reply:
This page is a placeholder for when the Java - Advanced curriculum is created.  Should I still add the metadata ?

Reply:
OK, never mind then. Just make sure that if you try to go to the relative URL /java-tictactoe/placeholder in the website, that you don't end up seeing this page...